src/com/owncloud/android/files/FileOperationsHelper.java

```diff
@@ -541,6 +543,17 @@ public void retryUpload(OCUpload upload) {
         }
     }
 
+    public void retryUploadsForAccount(Account account) {
```

Comment:
Could we move this method to ```FileUploader```? As a static method ```retryUploadsForAccount(Account account, Context context)```, similar to the new static methods to ease the creation of upload requests. Thanks.